### DISCUSSION TITLE
Help me !

### DISCUSSION THREAD

**Discussion by adxlm** [community/community#39103] [New to GitHub]
For my work, I need to generate a Github URL of my repository. Its suppose to be on "SETTING" and  "Github Pages" section but there isn't !
I don't find the "Github Pages" section, I don't know what I done wrong, please help me... thnks !

**Comment by radar-blip** [comment]
You might be in the wrong settings. Open your repo, `Settings` is at the top of that page, then in the left column there is `Pages`.
Like below:

![pagess](https://user-images.githubusercontent.com/39196767/201769666-acacb3b7-488f-4975-8522-505fb5453ded.png)

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @adxlm  , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
